redis/plural/recipes/aws-redis-cluster.yaml

```diff
@@ -1,4 +1,4 @@
-name: aws-redis
+name: aws-redis-cluster
```

Comment:
why are we renaming the recipes?

Reply:
Because the 2 different redis recipes have the same name but got overwritten in the API.